pkg/telemetryquerylanguage/contexts/tqllogs/logs.go

```diff
@@ -27,21 +27,21 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql"
 )
 
-type LogTransformContext struct {
+type TransformContext struct {
 	Log                  plog.LogRecord
```

Comment:
I would keep consistent and call this `LogRecord` not `Log`

Reply:
Done